.github/ISSUE_TEMPLATE/4-release_checklist.md

```diff
@@ -10,47 +10,50 @@ assignees: ''
-      and [Testing your install]((https://www.pygmt.org/dev/install.html#testing-your-install) regarding GMT-Ghostscript incompatibility
+- [ ] Update warnings in `pygmt/_show_versions.py` as well as notes in
+      [Not working transparency](https://www.pygmt.org/dev/install.html#not-working-transparency)
+      regarding GMT-Ghostscript incompatibility
 - [ ] Reserve a DOI on [Zenodo](https://zenodo.org) by clicking on "New Version"
-- [ ] Review the ["PyGMT Team" page](https://www.pygmt.org/dev/team.html)
-- [ ] Finish up 'Changelog entry for v0.x.x' Pull Request:
-  - [ ] Add a new entry in `doc/_static/version_switch.js` for documentation switcher
-  - [ ] Update `CITATION.cff` and BibTeX at https://github.com/GenericMappingTools/pygmt#citing-pygmt
-    - [ ] Update authorship list
-    - [ ] Update DOI (and url for BibTeX)
-    - [ ] Update version
-    - [ ] Update date released
-  - [ ] Add the documentation link `doc/minversions.md`
-  - [ ] Add minimum required version information `doc/minversions.md`
-  - [ ] Copy draft changelog from Release Drafter and edit it to look nice ([see maintainers guide for details](https://www.pygmt.org/dev/maintenance.html#updating-the-changelog))
+- [ ] Finish up the "Changelog entry for v0.x.x" Pull Request (Use the previous changelog PR as a reference)
+- [ ] Run `make codespell` to check common misspellings. If there are any, either fix them or add them to `ignore-words-list` in `pyproject.toml`
+- [ ] Draft the announcement on https://hackmd.io/@pygmt
 
 **Release**:
+
 - [ ] At the [PyGMT release page on GitHub](https://github.com/GenericMappingTools/pygmt/releases):
   - [ ] Edit the draft release notes with the finalized changelog
   - [ ] Set the tag version and release title to vX.Y.Z
+  - [ ] Make sure "Set as the latests release" and "Create a discussion for this release" are selected
   - [ ] Make a release by clicking the 'Publish Release' button, this will automatically create a tag too
-- [ ] Manually upload the pygmt-vX.Y.Z.zip and baseline-images.zip files to https://zenodo.org/deposit, ensure that it is filed under the correct reserved DOI
+- [ ] Download the pygmt-X.Y.Z.zip and baseline-images.zip from the release page,
+      rename pygmt-X.Y.Z.zip to pygmt-vX.Y.Z.zip, and upload the two zip files to
+      https://zenodo.org/deposit, ensure that they are filed under the correct reserved DOI
 
 **After release**:
-- [ ] Update conda-forge [pygmt-feedstock](https://github.com/conda-forge/pygmt-feedstock) [Done automatically by conda-forge's bot, but remember to pin SPEC0 versions]
+
+- [ ] Update conda-forge [pygmt-feedstock](https://github.com/conda-forge/pygmt-feedstock)
+      [Done automatically by conda-forge's bot, but can also be triggered manually by opening an issue with the title `@conda-forge-admin, please update version`. Remember to pin SPEC0 versions]
```

Comment:
Need to remember to manually update the Python version pin, xref https://github.com/conda-forge/pygmt-feedstock/pull/28#issuecomment-1703802989

```suggestion
      [Done automatically by conda-forge's bot. Remember to pin Python and SPEC0 versions]
```